cmd/up/up.go

```diff
@@ -903,3 +921,19 @@ func setBuildEnvVars(ctx context.Context, m *model.Manifest) error {
 	}
 	return builder.Build(ctx, buildOptions)
 }
+
+// wakeNamespaceIfApplies wakes the namespace if it is sleeping
+func wakeNamespaceIfApplies(ctx context.Context, ns string, k8sClient kubernetes.Interface, okClient types.OktetoInterface) error {
+	n, err := k8sClient.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// If the namespace is not sleeping, do nothing
+	if n.Labels["space.okteto.com/status"] != "Sleeping" {
```

Comment:
should we use a constant for this label?

Reply:
yeah, I can move it to a constant

Reply:
I saw that there are several places where we have hardcoded this value so we will need to call the new constant in them as well

Reply:
Changed in all the places